api/workAllocation/util.ts

```diff
@@ -39,37 +37,3 @@ export function prepareCaseWorkerForService(baseUrl: string, serviceId: string)
 export function prepareCaseWorkerForLocationAndService(baseUrl: string, locationId: string, serviceId: string) {
   return `${baseUrl}/caseworker/location/${locationId}/service/${serviceId}`
 }
-
```

Comment:
Errr... why has this been removed? This is needed. It won't even compile with this method removed.